apps/elixir_ls_debugger/lib/debugger/server.ex

```diff
@@ -400,6 +403,30 @@ defmodule ElixirLS.Debugger.Server do
     end
   end
 
+  defp all_variables(paused_processes) do
+    paused_processes
```

Comment:
You can combine some of the operations to reduce the number of iterations and apply filtering berore mapping
```
paused_processes
|> Enum.flat_map(fn {_pid, paused_process} -> paused_process.frames |> Map.values end)
|> Enum.filter(& match?(%Frame{bindings: bindings} when is_map(bindings), &1))
|> Enum.flat_map(fn %Frame{bindings: bindings} -> bindings |> Enum.map(& rename_binding_to_classic_variable/1))
```
```
for {_pid, paused_process} <- paused_processes,
{_key, values} <- paused_process.frames,
%Frame{bindings: bindings} when is_map(bindings) <- values,
binding <- bindings,
do: rename_binding_to_classic_variable(binding)
```